.gitignore

```diff
@@ -0,0 +1,20 @@
+#################################################################
+# Global/Vim.gitignore
+#################################################################
+# Swap
+[._]*.s[a-v][a-z]
+[._]*.sw[a-p]
+[._]s[a-rt-v][a-z]
+[._]ss[a-gi-z]
+[._]sw[a-p]
```

Comment:
Why is that needed in this repository?

Reply:
Yeah, I'm curious too.

Reply:
Hmm, good question! tbh, just added the gitignore from GitHub's gitignore templates: https://github.com/github/gitignore/blob/master/Global/Vim.gitignore

If you'd prefer shortening the gitignore to include only the patterns we fully understand, I'm on board with that.